src/satellite/process.ts

```diff
@@ -489,26 +717,39 @@ export class SatelliteProcess implements Satellite {
           incomingChanges.optype
         )
 
-        Object.assign(incomingChanges, { changes, optype })
+        /*
+        console.log('merge_op_tag')
+        console.log(localChanges)
+        console.log(incomingChanges)
+         */
+        const clear_tags = mergeOpTags(localChanges, incomingChanges)
```

Comment:
Somehow I think you're doing the right thing, but I do not understand why this is called clear_tags and is making a lot of confusion to me (as of my previous long comment). Let's discuss this again.